FirebasePerformance/README.md

```diff
@@ -1,10 +1,9 @@
 # Firebase Performance
 
-Firebase Performance is a free mobile app performance analytics service. It
+[Firebase Performance](https://firebase.google.com/docs/perf-mon) is a free mobile app performance analytics service. It
```

Comment:
`Firebase Performance Monitoring` is the full product name (I previously received tickets from developers who thought that Fireperf is the performance about Firestore)

Reply:
Sounds good! Fixed.